README.md

```diff
@@ -1,4 +1,4 @@
-# file-size
+# file-size [![Maven Central](https://img.shields.io/maven-central/v/me.saket.filesize/filesize?label=%20&color=success)](https://central.sonatype.com/artifact/me.saket.filesize/filesize) [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/me.saket.filesize/filesize?label=%20&color=lightgrey&server=https%3A%2F%2Foss.sonatype.org%2F)](https://oss.sonatype.org/content/repositories/snapshots/me/saket/filesize/filesize/)
```

Comment:
Might be better to place badges under the title, and separate them in each line.

And we can add

- `[![License](https://img.shields.io/github/license/saket/file-size)](LICENSE.txt)`
- `[![CI](https://github.com/saket/file-size/actions/workflows/test_and_release_snapshot.yml/badge.svg?branch=trunk&event=push)](https://github.com/saket/file-size/actions/workflows/test_and_release_snapshot.yml?query=event:push+branch:trunk)`